docs/docs-content/release-notes/release-notes.md

```diff
@@ -24,6 +26,145 @@ tags: ["release-notes"]
   [Validate Environment](../automation/palette-cli/commands/ec.md#validate-environment) section of the Palette EC
   command documentation to learn more.
 
+- `nodeDrainTimeout` for clusters has been changed from 10 mins from the previous version to infinite.
+
+  - New clusters will have an infinite `nodeDrainTimeout` upon creation.
+  - Existing clusters will retain their existing `nodeDrainTimeout` until there is a new machine created in the control
```

Comment:
Machines created in the control plane only or also worker nodes? I think worker nodes too.